client/src/Components/ShortLinks/ShortLinksList.jsx

```diff
@@ -32,7 +32,7 @@ const ShortLinkList = () => {
     });
   }, [shortLinkList]);
 
-  return <Fragment>{shortLinks ? shortLinks : 'Loading State Here'}</Fragment>;
+  return <Fragment>{shortLinks ? shortLinks : ''}</Fragment>;
```

Comment:
we should probably keep this loading text, to show it's loading, but maybe we will change to something which looks better in future

Reply:
Just remembered that I could download the photo from Figma. At least I practiced something with Photoshop. I also put back the loading state here text .